CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+<a name="2.0.0-rc.12"></a>
+
+# [2.0.0-rc.12](https://github.com/commercetools/ui-kit/compare/2.0.0-rc.11...2.0.0-rc.12) (2018-10-29)
+
+### Bug Fixes
+
+- import vars correctly ([#188](https://github.com/commercetools/ui-kit/issues/188))
+- remove overflow:none from body reset ([#189](https://github.com/commercetools/ui-kit/issues/189))
+- replace styled-components with react-emotion ([#184](https://github.com/commercetools/ui-kit/issues/184))
+- `SelectInput`: align indicator icons properly ([#190](https://github.com/commercetools/ui-kit/issues/190))
+
+### Changes
+
+- explicitly peer deep deps ([#156](https://github.com/commercetools/ui-kit/issues/156))
+- `bundling`: lower bundle size by discarding comments ([#173](https://github.com/commercetools/ui-kit/issues/173))
+- self contain styles first step ([#186](https://github.com/commercetools/ui-kit/issues/186))
+- `Text`: add `title` prop to all typography components ([#191](https://github.com/commercetools/ui-kit/issues/191))
+
+### BREAKING CHANGES
```

Comment:
We should provide as much info as possible about possible migration steps that needs to be done based on the breaking changes.

Reply:
Yeah, we just need to provide a bit of info like in our current readme. Something like

The old materials css files (colors.mod.css, spacings.mod.css) from the @commercetools-frontend/ui-kit/materials directory have been removed and replaced by a single exported css file `@commercetools-frontend/ui-kit/materials/custom-properties.css `. This can either be used directly inside a css file with the @import syntax, or it can be included through the `importFrom` property of `postcssCustomProperties` during your build step.